rules/linux/auditd/lnx_auditd_clean_disable_dmesg_logs_via_syslog.yml

```
@@ -0,0 +1,40 @@
+title: Clear Or Disable Dmesg Logs via Syslog Syscall - Linux
```

```suggestion
title: Clear or Disable Kernel Ring Buffer Logs via Syslog Syscall
```
I think it makes sense to add Kernel Ring Buffer Logs rather than dmseg logs because as i understand it we are looking for logs disable via syslog syscall and dmseg is tool/way to do it.